mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -107,6 +108,14 @@ public FakeMailContext build() {
 
     public static class SentMail {
 
+        private static MimeMessage tryCopyMimeMessage(MimeMessage msg) throws MessagingException {
+            try {
+                return new MimeMessage(msg);
+            } catch (Exception e) {
+                return MimeMessageBuilder.mimeMessageBuilder().build();
```

Comment:
I can't understand this part.

Reply:
A null message generated an exception upon the copy. I provided a default value. But

```
Optional.ofNullable(msg)
    .map(MimeMessage::new)
    .orElse(null);
```

Indeed seems better. We should rather use it.